esx_service/vmdk_ops.py

```diff
@@ -252,6 +252,7 @@ def cloneVMDK(vm_name, vmdk_path, opts={}, vm_uuid=None, vm_datastore=None):
         return err("Failed to initialize source volume path {0}: {1}".format(src_path, errMsg))
 
     src_vmdk_path = vmdk_utils.get_vmdk_path(src_path, src_volume)
+    logging.debug("cloneVMDK: src_path=%s, src_volume=%s, src_vmdk_path=%s", src_path, src_volume, src_vmdk_path)
```

Comment:
Yes. This is not related to FALLBACK feature. The reason I add this debug log is I found it is very difficult when I debug FALLBACK feature, the unit test failed in cloneVMDK related test. I think the log is helpful for others to debug if they hit errors in this part.